samples/Samples.SqlServer/Properties/launchSettings.json

```diff
@@ -12,7 +12,8 @@
         "CORECLR_PROFILER_PATH": "$(ProjectDir)$(OutputPath)profiler-lib\\Datadog.Trace.ClrProfiler.Native.dll",
 
         "DD_DOTNET_TRACER_HOME": "$(ProjectDir)$(OutputPath)profiler-lib",
-        "DD_INTEGRATIONS": "$(ProjectDir)$(OutputPath)profiler-lib\\integrations.json"
+        "DD_INTEGRATIONS": "$(ProjectDir)$(OutputPath)profiler-lib\\integrations.json",
+        "DD_PROFILER_EXCLUDE_PROCESSES": "sqlservr.exe"
```

Comment:
Is this is a typo? Good call on excluding SqlServer, since that just creates noise when the profiler can't attach to that

Reply:
You mean `sqlservr`? Nope, that's the name of the SQL Server process. Reminds me of the "8.3" filename limit back in the day.

Reply:
Yeah the `sqlservr`. Huh, I learned something new today! Cool thanks for clarifying 👍